src/components/reusable/details/details.ts

```diff
@@ -0,0 +1,75 @@
+import { LitElement, html } from 'lit';
+import { customElement, property, state, query } from 'lit/decorators.js';
+import Styles from './details.scss';
+
+/**
+ * Example component template.
+ * @fires on-click - Captures the click event and emits a custom event.
+ * @slot unnamed - Slot for child components.
+ */
+@customElement('kyn-details')
+export class Details extends LitElement {
+  static override styles = Styles;
+
+  /** exposed reactive string property */
+  @property({ type: String })
+  summary = '';
+
+  /** exposed reactive array property */
+  @property({ type: Array })
+  details: Array<any> = [];
```

Comment:
The details would probably work better as a `<slot>` in this case. Generally using slots allows us to be more flexible, and give more control to the application.